config/locales/client.en.yml

```diff
@@ -15,6 +15,8 @@ en:
       import:
         label: "Import"
         modal: "Import A Query"
+        unparseable_json: "Unparseable json file."
+        wrong_json: "Wrong json file. A json file should contain an 'query' object, which should at least has a 'sql' property."
```

Comment:
Nitpick, should be `"which should at least have an 'sql' property"`. Also...I don't mind if you change this or not, it's just another nitpick, but I think JSON should be capitalised because it is an acronym

Reply:
I've changed both, thank you 👍, please feel free to comment such mistakes.